tests/arch_tests.cmake

```diff
@@ -104,50 +106,54 @@ function(add_xc7_test)
     add_custom_target(${arch}-${test_name}-dcp DEPENDS ${dcp})
 
     # Bitstream generation target from DCP
-    set(dcp_vivado_tcl ${CMAKE_SOURCE_DIR}/tests/common/dcp_vivado.tcl)
-    set(dcp_bit ${output_dir}/${name}.dcp.bit)
-    add_custom_command(
-        OUTPUT ${dcp_bit}
-        COMMAND ${CMAKE_COMMAND} -E env
-            VIVADO_SETTINGS=${VIVADO_SETTINGS}
-            OUTPUT_DIR=${output_dir}
-            DCP_FILE=${dcp}
-            BIT_FILE=${dcp_bit}
-            ${quiet_cmd}
-            ${run_vivado} -mode batch -source ${dcp_vivado_tcl}
-        DEPENDS
-            ${arch}-${test_name}-dcp
-            ${run_vivado}
-            ${dcp_vivado_tcl}
-            ${dcp}
-    )
-
-    add_custom_target(${arch}-${test_name}-dcp-bit DEPENDS ${dcp_bit})
-    add_dependencies(all-vendor-bit-tests ${arch}-${test_name}-dcp-bit)
-    add_dependencies(all-${device}-vendor-bit-tests ${arch}-${test_name}-dcp-bit)
+    if (NOT no_bitstream)
```

Comment:
I am thinking whether it would be better to exclude only the dependency addition of the bitstream generation (and FASM) to the `all-tests...` targets.

Reply:
But then we will have broken targets. I don't think it makes sens to add them if they won't work.